### PR TITLE
Update links to k8s control plane and k8s objects on controller page

### PR DESCRIPTION
**Pull request by Surbowl**
Fix links 😀

### CODE REVIEW

File: content/zh/docs/concepts/overview/working-with-objects/kubernetes-objects.md [hunk 1]
```diff
@@ -195,7 +195,7 @@ and the `spec` format for a `Deployment` can be found
 * Learn about the most important basic Kubernetes objects, such as [Pod](/docs/concepts/workloads/pods/pod-overview/).
 * Learn about [controllers](/docs/concepts/architecture/controller/) in Kubernetes
 -->
-* [Kubernetes API 概述](/zh/docs/reference/using-api/api-overview/) 提供关于 API 概念的进一步阐述
+* [Kubernetes API 总览](/zh/docs/reference/using-api/api-overview/) 提供关于 API 概念的进一步阐述
 * 了解最重要的 Kubernetes 基本对象，例如 [Pod](/zh/docs/concepts/workloads/pods/)
-* 了解 Kubernetes 中的[控制器](/zh/docs/concepts/architecture/controller/)
+* 了解 Kubernetes 中的 [控制器](/zh/docs/concepts/architecture/controller/)
```
Comment (tengqm) [inline content/zh/docs/concepts/overview/working-with-objects/kubernetes-objects.md:200]: nit: this extra space is not needed

File: content/zh/docs/concepts/overview/working-with-objects/kubernetes-objects.md [hunk 2]
```diff
@@ -55,7 +55,7 @@ Kubernetes 对象是 “目标性记录” —— 一旦创建对象，Kubernete
 [Kubernetes API](/zh/docs/concepts/overview/kubernetes-api)。
 比如，当使用 `kubectl` 命令行接口时，CLI 会执行必要的 Kubernetes API 调用，
 也可以在程序中使用
-[客户端库](/zh/docs/reference/using-api/client-libraries/)直接调用 Kubernetes API。
+[客户端库](/zh/docs/reference/using-api/client-libraries/) 直接调用 Kubernetes API。
```
Comment (tengqm) [inline content/zh/docs/concepts/overview/working-with-objects/kubernetes-objects.md:58]: this extra link is not needed

**Comment by tengqm** [comment]
@Surbowl In future PRs please keep a single commit.

**Comment by Surbowl** [comment]
> @Surbowl In future PRs please keep a single commit.

I got it, Thanks ! 😀

**Pull request merged**